build.gradle

```diff
@@ -128,7 +128,7 @@ def versions = [
         serviceTokenGenerator   : '3.1.1',
         springfoxSwagger        : '2.9.2',
         sonarPitest             : '0.5',
-        tomcat                  : '9.0.35',
+        tomcat                  : '9.0.36',
```

Comment:
required due to 
[2020-07-03T13:24:10.156Z] Found 2 vulnerabilities in project finrem-case-orchestration
[2020-07-03T13:24:10.156Z] One or more dependencies were identified with known vulnerabilities in finrem-case-orchestration:
[2020-07-03T13:24:10.156Z] tomcat-embed-core-9.0.35.jar (pkg:maven/org.apache.tomcat.embed/tomcat-embed-core@9.0.35, cpe:2.3:a:apache:tomcat:9.0.35:*:*:*:*:*:*:*, cpe:2.3:a:apache_software_foundation:tomcat:9.0.35:*:*:*:*:*:*:*, cpe:2.3:a:apache_tomcat:apache_tomcat:9.0.35:*:*:*:*:*:*:*) : CVE-2020-11996
[2020-07-03T13:24:10.156Z] tomcat-embed-websocket-9.0.35.jar (pkg:maven/org.apache.tomcat.embed/tomcat-embed-websocket@9.0.35, cpe:2.3:a:apache:tomcat:9.0.35:*:*:*:*:*:*:*, cpe:2.3:a:apache_software_foundation:tomcat:9.0.35:*:*:*:*:*:*:*, cpe:2.3:a:apache_tomcat:apache_tomcat:9.0.35:*:*:*:*:*:*:*, cpe:2.3:a:java-websocket_project:java-websocket:9.0.35:*:*:*:*:*:*:*) : CVE-2020-11996